_episodes_rmd/06-best-practices-R.Rmd

```diff
@@ -3,11 +3,14 @@ title: "Best Practices for Writing R"
 teaching: 10
 exercises: 0
 questions:
-- "How can I write R that other people can understand and use?"
+- "What are the best practices for writing R so others can understand what I have done?"
 objectives:
-- "Define best formatting practices when writing code in R scripts."
-- "Synthesize a consistent personal coding style to increase code readability, consistency, and repeatability."
-- "Apply this style to your own code."
+- "Describe changes made to code for future reference."
+- "Understand the importance about requirements and dependencies for your code."
+- "Know when to use setwd()."
+- "To identify and segregate distinct components in your code using # or #-."
+- "Additional best practice recommendations."
+
```

Comment:
These updates increase the match of objectives to the actual content, so +1 for this :-)

Reply:
+1 from me too :)